FatFs_SPI/sd_driver/sd_card.c

```diff
@@ -644,7 +653,7 @@ static uint32_t sd_go_idle_state(sd_card_t *pSD) {
      * not be interpreted as a command and get lost. For some cards retrying
      * the command overcomes this situation. */
     for (int i = 0; i < SD_CMD0_GO_IDLE_STATE_RETRIES; i++) {
-        sd_cmd(pSD, CMD0_GO_IDLE_STATE, 0x0, false, &response);
+        sd_cmd(pSD, CMD0_GO_IDLE_STATE, 0x0, false, &response, NULL);
```

Comment:
Thanks! Can you change the destination branch to `dev` instead of `master`? `dev` is for new development, so it can be integrated and tested before merging it into `master`. Also, there are the parallel `sdio` and `dev_sdio` branches. `master` and `dev` are more or less frozen and anything new that I do goes into `sdio`. So, it would be nice if you could also make this available for `dev_sdio`.

Reply:
Yes, of course! I will address these comments and get PRs setup for those dev branches. Thanks

Reply:
Also, consider adding something to `README.md` **Notes about Card Detect** explaining how to use this.